examples/aggregator/gcp/pod.yaml

```diff
@@ -24,6 +27,26 @@ spec:
       env:
         - name: RUST_LOG
           value: info
+    - name: prometheus-to-sd
```

Comment:
Could you add a comment here? in particular, the `sd` part of the name is quite obscure. The link you have in the commit message should probably be here instead.